### ISSUE TITLE
Paging Error

### ISSUE THREAD

**Issue by Forsakencrow** [tullamods/Dominos#437]
Hello,

First time commenting so forgive me if I forget something.

Love the addon but recently I've been experiencing a weird error where some of my paging does not work. For example I will have my second action bar setup to be paged to whenever I hold down ctrl + the associated button. While the toolbar will change to reflect the paging pressing ctril + the other button is not responsive.

So say I have my taunt on page 2, key 1. So normally I'd hit ctrl + 1 to taunt, but whenever I press this there is no response. However, if I hold down ctrl and manually move my mouse over the icon and click it then it works.

This is not limited to the ctrl binding either, as sometimes it happens to my shift binding too. I have no idea what to do and even moving my buttons to a different action bar (action bar 5) and trying to use them still does nothing. So is there anything that I can do about this error or am I stuck with not using those buttons whenever they experience this error?

Thanks in advance for any help you can provide.

**Comment by Tuller** [comment]
Can you  double check what you have for bindings on the Blizzard end? You may want to clear them for shift + 1,2,3,etc as they can take priority over the paging stuff for Dominos

**Comment by Forsakencrow** [comment]
> Can you double check what you have for bindings on the Blizzard end? You may want to clear them for shift + 1,2,3,etc as they can take priority over the paging stuff for Dominos

I checked my bindings on the Blizzard end and made sure that they were clear and still no luck. But a strange thing is the paging can work and the first 2 keys but not the rest of the bar. For example I can have keys setup like: ctrl + 1 and ctrl + 2 work, but ctrl + 3, 4, etc will not work.

**Comment by Tuller** [comment]
That makes me still think that ctrl + 3 etc are bound to something else somewhere.

**Comment by Forsakencrow** [comment]
If it helps any this issue only occurs in retail, it does not occur in classic. I have the same setup in classic and it works fine without any issue.

**Comment by Tuller** [comment]
Here's what looks to be the issue:
```lua
{
  ["showstates"] = "[extrabar]show;hide",
}
```
Try this on the profile:
```
/run Dominos.Frame:Get('extra'):SetShowStates(nil)
```

I'll throw this into the next release.

**Comment by Tuller** [comment]
This is one of those things that I'm going to flag as "probably fixed in the 9.0 release"
